### ISSUE TITLE
sending packets at a given speed

### ISSUE THREAD

**Issue by fsuarezo** [enclave-networks/multicast-test#4]
Testing for different systems on different environments, sometimes it's useful to see if the system can handle a bit more throughput, so implementing a "send xxx packages per second" would be cool.

**Comment by enclave-marc-barry** [comment]
Agreed. Are you planning to raise a PR?

**Comment by fsuarezo** [comment]
No, I'm not a developer, sorry. I'm only hoping that someone can do the trick.
